documentation/trouble_kahadb_corruption.markdown

```diff
@@ -20,8 +20,14 @@ Why does corruption occur?
 In some cases this database may corrupt. Lots of things may cause this:
```

Comment:
... may corrupt ... "or become unreadable by our software." its not a corruption afaik, lets make it clear.